app/models/miq_schedule.rb

```diff
@@ -217,6 +217,12 @@ def action_check_compliance(obj, _at)
     _log.info("Action [#{name}] has been run for target type: [#{obj.class}] with name: [#{obj.name}]")
   end
 
+  def action_automation_request(_klass, _at)
```

Comment:
@bdunne - Per - https://github.com/syncrou/manageiq/blob/dc3578035729515795876b3f785c8b4919c2233b/app/models/miq_schedule.rb#L100

The `send` method is always passing 2 parameters.  The other `action_` methods are doing the same with unused parameters depending on the need.  

I was planning on refactoring how the `send` makes a decision when sending parameters but was thinking I would do that in another PR.